config/locales/sv.yml

```diff
@@ -0,0 +1,643 @@
+    contact: 'Kontakta Oss'
+    contact_description: "<p>Vänligen %{email_link} om du har några frågor eller problem.</p><p class=\"no_margin_bottom\"><em>Sekretesspolicy anpassad från %{jamie_king_media}.</em></p>"
+  resources:
+    description: "Välkommen till våra kurerade psykiska resurser som organiseras av %{communities}, %{education}, %{hotlines}, och %{services}."
+    communities: Gemenskaperna
+    education: Utbildning
+    hotlines: Direktlinje
+    services: Tjänster
+    tags:
+      people_of_colour: människor av färg
+      women_of_colour: kvinnor av färg
+      black_women: svarta kvinnor
+      domestic_violence: våld i hemmet
+      dating_violence: dejting våld
+      cyberbullying: nätmobbning
+      therapy: terapi
+      counseling: rådgivning
+      youth: ungdom
+      lgbtqia: LGBTQIA
+      harassment: trakasserier
+      self_care: egenvård
+      women: kvinnor
+      chatbot: chatbot
+      anonymous: anonym
+      free: fri
+      paid: betald
+      twenty_four_seven: 24/7
+      videos: videoklipp
+      blog: blogg
+      podcast: podcast
+      books: böcker
+      publication: offentliggörande
+      video_chat: videochatt
+      texting: textning
+      open_source: öppen källa
+      workplace: arbetsplats
+      social_justice: social rättvisa
+      tech_industry: teknisk industri
+      meditation: meditation
+      social_anxiety: social ångest
+      trigger_warnings: utlösande varningar
+      netflix: netflix
+      homeless: hemlös
+      talks: samtal
+      voice_chat: röstchatt
+      training: utbildning
+      directory: katalog
+      self_injury: självskada
+      treatment: behandling
+      care_planning: vårdplanering
+      depression: depression
+      ocd: OCD
+      advice: råd
+      forum: forum
+      learning: inlärning
+      crisis_prevention: krishantering
+      safety_planning: säkerhetsplanering
+      suicide_prevention: förebyggande av självmord
+      android: android
+      ios: iOS
+      support_groups: stödgrupper
+      mindfulness: mindfulness
+      journaling: journalföring
+      mood_tracking: humörspårning
+
+ profile:
+  index:
+   title: 'Profil'
+   edit_user: 'Anpassa profilen'
+
+ pusher:
+  not_authorized: 'Inte behörig'
+
+ search:
+  search: 'Sök'
+  search_by_name: 'Sök på namn'
+  form:
+   search_by_location: 'Sök efter plats'
+   search_by_email: 'Sök via e-mail'
+
+  index:
+   title: 'Sök efter Allierade'
+   user_not_found: 'Användaren %{email} hittades inte'
+
+ shared:
+  footer:
+    licence_name: 'AGPL-3.0'
+    license: 'Vi &hearts; FOSS'
+    foss: 'Fri och öppen källa programvara'
+    license_subtitle: 'Licensierad under %{license}'
+    connect: 'ansluta'
```

Comment:
Q: the `connect`-button in the footer - what does it do? I can't figure this one out, but the translation seems off (because it might be about "Contact us" more than "Create connection").

Reply:
It's a title for social media connections
<img width="827" alt="screen shot 2017-09-13 at 7 49 12 pm" src="https://user-images.githubusercontent.com/8427242/30392305-b520dca0-98bc-11e7-9241-0f93c762bf7d.png">